Makefile

```diff
@@ -134,6 +134,21 @@ else
 	go test -v -timeout 3600s $(TEST_ARGS) ./test/e2e/...
 endif
 
+kind-e2e-test: build kustomize docker-build create-kind-multicluster kind-load-image-multi e2e-test
+
+kind-setup: build kustomize docker-build create-kind-cluster kind-load-image cert-manager
```

Comment:
But that's what it does, the k8ssandra cluster isn't created here. We're just setting up kind so that a k8ssandra cluster can be created.

Reply:
Ah, OK, so looking at the README it says 

> Run `make kind-setup` to create a single kind cluster and deploy k8ssandra-operator along with its dependencies. 

So maybe it is just the README that needs an update?

Reply:
Also I want to mention I get an error with `make kind-setup` (just tested) dumping output here:

```
/Users/milesgarnsey/projects/k8ssandra-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
go build -o bin/manager main.go
docker buildx build --load -t k8ssandra/k8ssandra-operator:latest .
[+] Building 237.0s (19/19) FINISHED                                                                                                                        
 => [internal] load build definition from Dockerfile                                                                                                   0.0s
 => => transferring dockerfile: 844B                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                      0.0s
 => => transferring context: 193B                                                                                                                      0.0s
 => [internal] load metadata for docker.io/library/golang:1.16                                                                                         4.5s
 => [internal] load metadata for docker.io/library/busybox:latest                                                                                      4.2s
 => [builder  1/10] FROM docker.io/library/golang:1.16@sha256:0056b049979bfcf13ac2ede60b810349396fab1d510cb60701503dccd01f9153                        38.0s
 => => resolve docker.io/library/golang:1.16@sha256:0056b049979bfcf13ac2ede60b810349396fab1d510cb60701503dccd01f9153                                   0.0s
 => => sha256:ba66844146b358c9ba2badd0d1daaacdbf1e1b93b85bb15f9c634cd116edb6a9 155B / 155B                                                             0.3s
 => => sha256:261c19c9b43ea49ddb276e79dd61e8b1b0a3401e89ea85c37b5eb38194c2e20e 129.03MB / 129.03MB                                                    30.6s
 => => sha256:4e94c8ba5874feacb79d2235c4b275408a5110a5c12fd4c45200199ad8847091 85.72MB / 85.72MB                                                      27.5s
 => => sha256:27b0a22ee906271a6ce9ddd1754fdd7d3b59078e0b57b6cc054c7ed7ac301587 54.57MB / 54.57MB                                                      17.2s
 => => sha256:911ea9f2bd51e53a455297e0631e18a72a86d7e2c8e1807176e80f991bde5d64 10.87MB / 10.87MB                                                       5.4s
 => => sha256:2756ef5f69a5190f4308619e0f446d95f5515eef4a814dbad0bcebbbbc7b25a8 5.15MB / 5.15MB                                                         3.4s
 => => sha256:955615a668ce169f8a1443fc6b6e6215f43fe0babfb4790712a2d3171f34d366 54.93MB / 54.93MB                                                      17.7s
 => => extracting sha256:955615a668ce169f8a1443fc6b6e6215f43fe0babfb4790712a2d3171f34d366                                                              2.3s
 => => extracting sha256:2756ef5f69a5190f4308619e0f446d95f5515eef4a814dbad0bcebbbbc7b25a8                                                              0.3s
 => => extracting sha256:911ea9f2bd51e53a455297e0631e18a72a86d7e2c8e1807176e80f991bde5d64                                                              0.2s
 => => extracting sha256:27b0a22ee906271a6ce9ddd1754fdd7d3b59078e0b57b6cc054c7ed7ac301587                                                              2.2s
 => => extracting sha256:4e94c8ba5874feacb79d2235c4b275408a5110a5c12fd4c45200199ad8847091                                                              2.0s
 => => extracting sha256:261c19c9b43ea49ddb276e79dd61e8b1b0a3401e89ea85c37b5eb38194c2e20e                                                              4.1s
 => => extracting sha256:ba66844146b358c9ba2badd0d1daaacdbf1e1b93b85bb15f9c634cd116edb6a9                                                              0.0s
 => [internal] load build context                                                                                                                      6.0s
 => => transferring context: 472.06kB                                                                                                                  6.0s
 => [stage-1 1/3] FROM docker.io/library/busybox@sha256:52f73a0a43a16cf37cd0720c90887ce972fe60ee06a687ee71fb93a7ca601df7                               0.5s
 => => resolve docker.io/library/busybox@sha256:52f73a0a43a16cf37cd0720c90887ce972fe60ee06a687ee71fb93a7ca601df7                                       0.0s
 => => sha256:24fb2886d6f6c5d16481dd7608b47e78a8e92a13d6e64d87d57cb16d5f766d63 767.32kB / 767.32kB                                                     0.4s
 => => extracting sha256:24fb2886d6f6c5d16481dd7608b47e78a8e92a13d6e64d87d57cb16d5f766d63                                                              0.1s
 => [builder  2/10] WORKDIR /workspace                                                                                                                 3.8s
 => [builder  3/10] COPY go.mod go.mod                                                                                                                 0.0s
 => [builder  4/10] COPY go.sum go.sum                                                                                                                 0.0s
 => [builder  5/10] RUN go mod download                                                                                                              123.8s
 => [builder  6/10] COPY main.go main.go                                                                                                               1.7s
 => [builder  7/10] COPY api/ api/                                                                                                                     0.0s
 => [builder  8/10] COPY pkg/ pkg/                                                                                                                     0.0s
 => [builder  9/10] COPY controllers/ controllers/                                                                                                     0.0s
 => [builder 10/10] RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go                                                          57.6s
 => [stage-1 2/3] COPY --from=builder /workspace/manager .                                                                                             0.2s
 => exporting to oci image format                                                                                                                      6.5s
 => => exporting layers                                                                                                                                4.5s
 => => exporting manifest sha256:32136afe4de227af4a2c683984f6cad2dac494fc53969d116e988758af527939                                                      0.0s
 => => exporting config sha256:ccfad18e16d0d0eebaf0848b81a7ecf34605f9302d0eae72d0cdbf4010e7fd3e                                                        0.0s
 => => sending tarball                                                                                                                                 1.9s
 => importing to docker                                                                                                                                1.0s
scripts/setup-kind-multicluster.sh --clusters 1 --kind-worker-nodes 4
Deleting existing clusters...
Deleting cluster 1 out of 1
Deleting cluster "k8ssandra-0" ...

Creating clusters...
Creating cluster 1 out of 1
Creating cluster "k8ssandra-0" ...
 ✓ Ensuring node image (kindest/node:v1.21.2) 🖼 
 ✓ Preparing nodes 📦 📦 📦 📦 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✓ Joining worker nodes 🚜 
Set kubectl context to "kind-k8ssandra-0"
You can now use your cluster with:

kubectl cluster-info --context kind-k8ssandra-0

Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
Error response from daemon: endpoint with name kind-registry already exists in network kind

Generating kubeconfig
Generating in-cluster kubeconfig
kind load docker-image --name k8ssandra-0 k8ssandra/k8ssandra-operator:latest
Image: "k8ssandra/k8ssandra-operator:latest" with ID "sha256:ccfad18e16d0d0eebaf0848b81a7ecf34605f9302d0eae72d0cdbf4010e7fd3e" not yet present on node "k8ssandra-0-worker2", loading...
Image: "k8ssandra/k8ssandra-operator:latest" with ID "sha256:ccfad18e16d0d0eebaf0848b81a7ecf34605f9302d0eae72d0cdbf4010e7fd3e" not yet present on node "k8ssandra-0-worker3", loading...
Image: "k8ssandra/k8ssandra-operator:latest" with ID "sha256:ccfad18e16d0d0eebaf0848b81a7ecf34605f9302d0eae72d0cdbf4010e7fd3e" not yet present on node "k8ssandra-0-control-plane", loading...
Image: "k8ssandra/k8ssandra-operator:latest" with ID "sha256:ccfad18e16d0d0eebaf0848b81a7ecf34605f9302d0eae72d0cdbf4010e7fd3e" not yet present on node "k8ssandra-0-worker4", loading...
Image: "k8ssandra/k8ssandra-operator:latest" with ID "sha256:ccfad18e16d0d0eebaf0848b81a7ecf34605f9302d0eae72d0cdbf4010e7fd3e" not yet present on node "k8ssandra-0-worker", loading...
kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
customresourcedefinition.apiextensions.k8s.io/certificaterequests.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/certificates.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/challenges.acme.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/clusterissuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/issuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/orders.acme.cert-manager.io created
namespace/cert-manager created
serviceaccount/cert-manager-cainjector created
serviceaccount/cert-manager created
serviceaccount/cert-manager-webhook created
clusterrole.rbac.authorization.k8s.io/cert-manager-cainjector created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrole.rbac.authorization.k8s.io/cert-manager-view created
clusterrole.rbac.authorization.k8s.io/cert-manager-edit created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-approve:cert-manager-io created
clusterrole.rbac.authorization.k8s.io/cert-manager-webhook:subjectaccessreviews created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-cainjector created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-approve:cert-manager-io created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-webhook:subjectaccessreviews created
role.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
role.rbac.authorization.k8s.io/cert-manager:leaderelection created
role.rbac.authorization.k8s.io/cert-manager-webhook:dynamic-serving created
rolebinding.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager-webhook:dynamic-serving created
service/cert-manager created
service/cert-manager-webhook created
deployment.apps/cert-manager-cainjector created
deployment.apps/cert-manager created
deployment.apps/cert-manager-webhook created
mutatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
validatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
kubectl rollout status deployment cert-manager-webhook -n cert-manager
Waiting for deployment "cert-manager-webhook" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager-webhook" successfully rolled out
/Users/milesgarnsey/projects/k8ssandra-operator/bin/kustomize build scripts | kubectl apply -f -
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/Users/milesgarnsey/projects/k8ssandra-operator/scripts'
error: no objects passed to apply
make: *** [kind-setup] Error 1
```

Reply:
> Ah, OK, so looking at the README it says
> 
> > Run `make kind-setup` to create a single kind cluster and deploy k8ssandra-operator along with its dependencies.
> 
> So maybe it is just the README that needs an update?

Nope, it's actually right :) 
I'm keeping the target simple because it's less to type and remember for a command that's going to be called quite often.
And since it's specific to k8ssandra-operator, it doesn't need to be very explicit (it's targeted at k8ssandra-operator devs).

I can't think of something that's not super long which would be more accurate than `kind-setup` 🤔  Any concise suggestions? 🙂

Reply:
k8ssandra-bootstrap? k8ssandra-setup? k8ssandra-up? k8dra-up? Even just `up` would be ok.